.github/CODEOWNERS

```diff
@@ -0,0 +1,25 @@
+/pages.de/ @pixelcmtd
+/pages.de/ @marchersimon
+/pages.es/ @navarroaxel
+/pages.it/ @mebeim
+/pages.it/ @yutyo
+/pages.pl/ @acuteenvy
+/pages.pt_BR/ @waldyrious
+/pages.pt_PT/ @waldyrious
+/pages.ta/ @kbdharun
+/pages.tr/ @yutyo
+/pages.zh/ @blueskyson
+/pages.zh_TW/ @blueskyson
+
+/contributing-guides/*.de.md @pixelcmtd
+/contributing-guides/*.de.md @marchersimon
+/contributing-guides/*.es.md @navarroaxel
+/contributing-guides/*.it.md @mebeim
```

Comment:
You can chain multiple people on the same path, i.e.

```suggestion
/contributing-guides/*.it.md @mebeim @yutyo
```

Sources:

> The pattern is followed by one or more GitHub usernames or team names using the standard @username or @org/team-name format.
> 
> ```
> /scripts/ @doctocat @octocat
> ```
> — https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

Reply:
That's a great idea, I will update the file now.